src/components/ProofOfAddress/Guidance/style.css

```diff
@@ -0,0 +1,56 @@
+@import (less) "../../Theme/constants.css";
+
+.content {
+  position: relative;
+  margin: 0 40px 1rem;
+  flex: 1;
+  flex-direction: column;
+  display: flex;
+
+  @media (--small-viewport) {
+    margin: 0 24px 1rem;
+  }
+}
+
+.subTitle {
+  font-size: 14px;
+
+  &::before {
+    content: '';
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    background-image: url('assets/warningSmall.svg');
+    vertical-align: middle;
+    line-height: 1em;
+    margin-right: 5px;
+    margin-top: -2px;
+  }
+}
+
+.bolder {
+  font-weight: 600;
+}
+
+.makeSure {
+  color: #2C3E4F;
+  font-weight: 600;
+  margin-bottom: 1em;
+
+  @media (--small-viewport) {
+    text-align: left;
+  }
+}
+
+.image {
+  background-image: url('assets/doc.svg');
```

Comment:
Please bear in mind no translation support is needed for now. We might have to find a different approach instead of using a background SVG, when there is a real need to get translated copy.

Reply:
ok